.github/actions/go-test/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM golang:1.14
```

Comment:
well isn't the last number subject to change with security patches? Why are we pinning to a micro version?

Reply:
To be consistent with the rest of the repo (which is outdated). Ideally, updating the repo's go version should be changing 1 file in the repo that is referenced from several places (like BUILD_NUMBER), but we're not there yet.

Until then, we should be able to look at the last commit that updated a go version in the repo as a guide for which files to change the next time we update a go version. If we go by minor version in some places and patch version in others, we can't do that.

If you'd like to avoid having 1 more place to change the go version, maybe this Action can run the unit test service in [/tools/golang](https://github.com/apache/trafficcontrol/tree/82452be484/tools/golang).

Reply:
seems easier to just add `.2`